tool/src/main/java/org/wildfly/security/tool/VaultCommand.java

```diff
@@ -214,7 +214,7 @@ public void execute(String[] args) throws Exception {
             }
 
             if (keystorePassword == null) {
-                keystorePassword = prompt(false, ElytronToolMessages.msg.vaultPasswordPrompt(), true, ElytronToolMessages.msg.vaultPasswordPromptConfirm());
+                keystorePassword = prompt(false, ElytronToolMessages.msg.vaultPasswordPrompt(), false, ElytronToolMessages.msg.vaultPasswordPromptConfirm());
```

Comment:
Thanks for the PR @sujathamlak! Since we don't need to confirm the password, in addition to setting `confirm` to `false` as you have already done, we should also use `null` instead of `ElytronToolMessages.msg.vaultPasswordPromptConfirm()` (since we don't have any confirmation text).

Reply:
Got it!

Reply:
@sujathamlak Great question! Yes, since nothing else is making use of this message, we can remove it.